trunk/SUAVE/Methods/Noise/Boom/lift_equivalent_area.py

```diff
@@ -0,0 +1,98 @@
+## @ingroupMethods-Noise-Boom
+# lift_equivalent_area.py
+# 
+# Created:  Sep 2020, E. Botero
+# Modified: 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+import numpy as np
+
+from SUAVE.Methods.Aerodynamics.Common.Fidelity_Zero.Lift import VLM
+
+# ----------------------------------------------------------------------
+#   Equivalent Area from lift for Sonic Boom
+# ----------------------------------------------------------------------
+## @ingroupMethods-Noise-Boom
+def lift_equivalent_area(config,analyses,conditions):
+    """ This method calculates the lift equivalent area for a vehicle for sonic boom analysis.
+    
+        Assumptions:
+        X_locs is the location where lift values are taken on the x-axis
+        AE_x is the lift equivalent area
+        CP is the coefficient of pressure at the flight conditions
+        
+        
+        Source:
+        N/A
+        
+        Inputs:
+        config                                [vehicle]
+        conditions
+              .freestream.dynamic_pressure    [pascals]
+              .freestream.mach_number         [-]
+              .aerodynamics.angle_of_attack   [radians]
+        analyses.aerodynamics.settings        [needed for discretization]
+        config.total_length                   [m]
+
+        Outputs:
+        X_locs                                [m]
+        AE_x                                  [m^2]
+        CP                                    [pascals]
+        
+        Properties Used:
+        N/A
+    """       
+    
+    S            = config.reference_area
+    q            = conditions.freestream.dynamic_pressure
+    mach         = conditions.freestream.mach_number
+    aoa          = conditions.aerodynamics.angle_of_attack
+    settings     = analyses.aerodynamics.process.compute.lift.inviscid_wings.settings
+    length       = config.total_length
+    
+    CL, CDi, CM, CL_wing, CDi_wing, cl_y, cdi_y, alpha_i, CP, Velocity_Profile = VLM(conditions, settings, config)
+    
+    VD = analyses.aerodynamics.geometry.vortex_distribution
+    
+    areas      = VD.panel_areas
+    normal_vec = VD.normals
+    XC         = VD.XC
+    ZC         = VD.ZC
+    z_comp     = normal_vec[:,2]
+
+    # The 2 is used because the CP acts on both the top and bottom of the panel
```

Comment:
What 2?

Reply:
Addressed. The old VLM didn't multiply by 2 for the CP, the new one has the 2 in the VLM script already.